src/specfem2D/rules.mk

```diff
@@ -49,9 +49,12 @@ specfem2D_TARGETS = \
 	$E/xspecfem2D \
 	$(EMPTY_MACRO)
 
-
+# AB AB: Do not reorganize the following list !!
```

Comment:
this ordering will break if the compilation is done in parallel, like for example by `make -j4`. one would need to define code dependencies if they are critical for compilation. once the dependencies have been added, the ordering can again be alphabetical. 

i'll check later, it seems that the module only needs constants and specfem_par modules, thus the dependency would be already set by using the compilation rule for .spec objects. now it's just about adding dependencies for the files using the moving_source_par module.